examples/contrail-bundle-k8s-nested-mode.yaml

```diff
@@ -0,0 +1,155 @@
+# OpenStack Contrail must be configured for nested mode!
+#
+# The following link-local services should be created:
+#
+#   LL Service Name    Service IP   Service Port   Fabric IP   Fabric Port
+#   ----------------   ----------   ------------   ---------   -----------
+#   K8s-cni-to-agent   10.10.10.5   9091           127.0.0.1   9091
+#
+# NOTE: Here 10.10.10.5 is the Service IP that was chosen by user. This can be any unused IP in the cluster
+
+series: xenial
+services:
+
+# contrail-kubernetes
+
+  contrail-kubernetes-master:
+    charm: ../contrail-kubernetes-master
+    series: xenial
+    options:
+      image-tag: 'master-774'
+      nested_mode: true
+      cluster_project: "{'domain':'default-domain','project':'admin'}"
+      cluster_network: "{'domain':'default-domain','project':'admin','name':'juju-net'}"
```

Comment:
maybe it's better to leave just cluster_network and evaluate project from it?
i can't see scenario where these two can be different